CamMerrill/Shackelford
# Smaller trowel dirt particles

## Summary
Trowel dirt particle effects are now smaller, a slightly different color, and the velocities were changed to make them have a stronger gravitational force.

## Testing
1. Load the Lab7 map
1. **Make sure to build lighting beforehand!** (under the Build menu)
1. Use the trowel to dig and see that the particles appear to be aesthetically improved

## Notes
If we decide to make additional changes in the future, this plugin + UE4 make editing the effects pretty simple.
